contrib/test/ci/cri-o.spec

```diff
@@ -135,9 +135,11 @@ export GOPATH=%{buildroot}/%{gopath}:$(pwd)/Godeps/_workspace:%{gopath}
 %endif
 
 %post
+ln -sf %{_unitdir}/%{service_name}.service {_unitdir}/%{repo}.service
```

Comment:
There's a typo here:
`{_unitdir}/%{repo}.service` should be `%{_unitdir}/%{repo}.service`

Reply:
good catch!